### ISSUE TITLE
please merge the latest Net-Ping master to blead

### ISSUE THREAD

**Issue by wchristian** [Perl/perl5#17992]
This commit:

https://github.com/rurban/Net-Ping/commit/10e96ffd8a056ee32854ea3dcb89d99d9411ac67

Fixes the one smoke test fail in:

http://perl5.test-smoke.org/report/115777

Given the commit has been merged into master on Net-Ping's end, it should be trivial to also merge into blead.

**Comment by atoomic** [comment]
Reopened as we want the last status from the upstream repo

**Comment by atoomic** [comment]
Note Net-Ping status on master as I am writing
Which indicates that we are missing two commits, but only one is relevant for us as Makefile.PL is not versioned in perl repo.
So we have to cherry pick changes from `10e96ff` to be at master.

```
10e96ff  skip flaky port test on windows (#20) (5 weeks ago, Christian Walde (Mithaldu))  (HEAD -> master, upstream/master)
 t/450_service.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

9276547  Fix and improve the metadata structure (#19) (6 weeks ago, Tom Hukins)
 Makefile.PL | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

4c28644  Release 2.73 (5 months ago, Reini Urban)
 Changes         | 6 ++++++
 README          | 4 +++-
 lib/Net/Ping.pm | 2 +-
 3 files changed, 10 insertions(+), 2 deletions(-)
```
